source/draw/RasterYuv422Semipacked.ooc

```diff
@@ -82,7 +82,7 @@ RasterYuv422Semipacked: class extends RasterPacked {
 			index := (this buffer pointer + y * this stride + x * this bytesPerPixel) as ColorMonochrome* // U or V value
 			(index + 1)@ = ColorMonochrome new(value y)
 			(index - 2*(x % 2))@ = ColorMonochrome new(value u)
-			(index - 2*((x + 1) % 2))@ = ColorMonochrome new(value v)
+			(index + 2*((x + 1) % 2))@ = ColorMonochrome new(value v)
```

Comment:
According to the `operator[]` above it should be `+`. If you print memory addresses when settings pixel at `(0,0)` you can see that this writes to memory 2 bytes before the buffer base address.